src/http/tx-scheduler/Transactions.test.ts

```diff
@@ -0,0 +1,296 @@
+import { HttpClient } from '../client/http'
+import {
+  IDeployContractRequest,
+  IETHTransactionParams,
+  IHttpGETRequest,
+  IHttpPOSTRequest,
+  IJobResponse,
+  ILog,
+  IScheduleResponse,
+  ISearchRequest,
+  ISendRawRequest,
+  ISendTransactionRequest,
+  ISendTransferRequest,
+  ITransactionResponse
+} from '../types'
+
+import { TxSchedulerTransactions } from './Transactions'
+
+const mockJobLogRes: ILog = {
+  message: 'Job Created',
+  status: 'CREATED',
+  createdAt: new Date()
+}
+
+const mockJobRes: IJobResponse = {
+  uuid: 'jobUUID',
+  chainUUID: 'chainUUID',
+  logs: [mockJobLogRes],
+  transaction: {},
+  status: 'STARTED',
+  type: 'eth://ethereum/transaction',
+  createdAt: new Date()
+}
+
+const mockScheduleRes: IScheduleResponse = {
+  uuid: 'scheduleUUID',
+  jobs: [mockJobRes],
+  createdAt: new Date()
```

Comment:
if you really need to fill this field you should make it non dependent on real time